test/ctia/test_helpers/es.clj

```diff
@@ -14,13 +14,22 @@
                       index
                       mapping)))
 
+(defn close-client [{:keys [conn]}]
+  "if the connection is native, close the client to avoid oom"
+  (when (instance? org.elasticsearch.client.transport.TransportClient conn)
+    (.close conn)))
+
 (defn fixture-recreate-store-indexes [test]
   "walk through all the es stores delete and recreate each store index"
 
   (doseq [store-impls (vals @store/stores)]
     (doseq [{:keys [state]} store-impls]
       (recreate-state-index state)))
-  (test))
+  (test)
+
+  (doseq [store-impls (vals @store/stores)]
```

Comment:
@gbuisson fyi, in case you didn't know. This is equivalent:

```clj
(doseq [store-impls (vals @store/stores)
        {:keys [state]} store-impls]
  (close-client state))
```